src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const mainImage = document.querySelector('.gallery__img');
+const galleryList = document.querySelector('.gallery__list');
+
+galleryList.addEventListener('click', (e) => {
+  const link = e.target.closest('.list-item__link');
+
+  e.preventDefault();
```

Comment:
The call to `e.preventDefault()` should be placed after checking if `link` exists. Otherwise, it may prevent default behavior for clicks that are not on a `.list-item__link`.